VirtualClosetApp/www/templates/closet/newItem.html

```diff
@@ -1,31 +1,45 @@
-<ion-view view-title="newItem">
+<ion-view view-title="newItem" ng-controller="newitemCtrl">
   <ion-content>
     <style>h3 {
       height: 110px;
     }
     </style>
     <h3 align="center">Add new item</h3>
 
+    <img ng-show="imgURI !== undefined" ng-src="{{imgURI}}" id = "itemImg" ng-model="itemImg">
+    <img ng-show="imgURI === undefined" ng-src="http://placehold.it/250x250">
+    <button class="button button-full button-positive " ng-click="takePicture() ">Take Image</button><br/>
+
+
     <div class="card">
-      <label class ="item item-input">
-        <span class = "input-label">Brand</span>
-        <input type="text" placeholder="e.g. Nike">
+
+      <select ng-model="item._category">
+        <option>Tops</option>
+        <option>Pants</option>
+        <option>Shoes</option>
+        <option>Formal</option>
+        <option>Jackets</option>
+        <option>Accessories</option>
+      </select>
+
+      <label class ="item item-input" >
+        <span class = "input-label">Item name</span>
+        <input type="text" placeholder="whatever you like" id="itemName" ng-model="item._Name">
```

Comment:
Is there a more appropriate placeholder value? Something like "Item Name" perhaps.